tests/python/gaia-ui-tests/gaiatest/apps/email/regions/setup.py

```diff
@@ -182,7 +183,14 @@ def tap_next(self):
                 *self._account_prefs_next_locator))))
 
     def check_for_emails_interval(self, value):
-        self.marionette.execute_script('document.querySelector("[data-l10n-id = settings-check-every-5min]").value = "%s";' % value)
+        # The following pref change allows us to check the mail within 1 second or longer,
+        # rather than the default value of 100 seconds
+        # The UI data layer of the UI is changed, because the minimum check mail time value is 5 min,
+        # which is far too long to check for in a test. This allows us to check earlier
+        self.marionette.execute_script("""
+            SpecialPowers.setIntPref('dom.requestSync.minInterval', 1);
+            document.querySelector("[data-l10n-id = settings-check-every-5min]").value = '%s';
+        """ % value, special_powers=True)
```

Comment:
The dom.requestSync.minInterval pref is in seconds.
The change makes the check for the imap server to 20 seconds, it's basically fixing this test, which did the same.